README.md

```diff
@@ -1,2 +1,228 @@
 # @logdna/env-config
 
+> Node.js Package to define, document, and assert environment variables
```

Comment:
like the README :)

Reply:
I can't take credit for this one, but thanks!  It's a team effort.